### DISCUSSION TITLE
Motion detection error

### DISCUSSION THREAD

**Discussion by daroczics** [gopro/labs#652] [Q&A]
Hello!
I updated the Gopro lab on the 11 camera to the latest version. Unfortunately, it only takes one picture in motion detection mode, even if I turn on the repeat button. The older version (1.20) did not have this error. Does anyone know a solution?

**Comment by dnewman-gpsw** [comment]
Motion detection has been flaky with Photos, as it was designed for video.

For photo try using the new scripting, it is ultimately more powerful.
![New Project-11_29_2023, 09_00_43](https://github.com/gopro/labs/assets/10856084/2ee258e4-67a9-492e-aafd-58fe46729d02)

This is very simple setup works for photo
![image](https://github.com/gopro/labs/assets/10856084/b0339a8f-abee-4239-a2fd-a6373174fbe1)

**Comment by daroczics** [comment]
Thank you very much, it works!
Where is this programming feature available?

**Reply by dnewman-gpsw** [reply]
https://gopro.github.io/labs/build/

**Comment by daroczics** [comment]
Thank you, you saved my life!!!
